### ISSUE TITLE
"Unknown Error" for all sequences in CLI with geneMap (v0.7.5, v0.8.0-alpha.6)

### ISSUE THREAD

**Issue by fmaguire** [nextstrain/nextclade#252]
Receiving the ```Error: in sequence "Canada/BC_37_0-2/2020": Unknown error. Please note that this sequence
will not be included in the results.``` error for every sequence ingested into the 0.7.5 CLI versions (docker + direct npm install) using the supplied default `geneMap.json` file but same sequence file is successfully processed in older 0.4.0-beta.6 CLI version, via web interface, or when not using the `-g geneMap.json` option.

# Reproduction
Error happening with all input sequences I've tried, all of below were run with the attached single sequence fasta file [error.txt](https://github.com/nextstrain/nextclade/files/5496152/error.txt) (uploaded with .txt suffix to get around gtihub filetype restrictions).

## Invocations resulting in error
- With the npm install of current main release of the CLI tool (0.7.5):

```
>npm install --global @neherlab/nextclade
>nextclade --input-fasta error.fasta --output-json error_fasta.json -g nextclade/packages/web/src/algorithms/defaults/sars-cov-2/geneMap.json

Error: in sequence "Canada/BC_37_0-2/2020": Unknown error. Please note that this sequence will not be included in the results.
```

- Same error with the npm install of latest alpha (0.8.0-alpha.6):

```
>npm install --global @neherlab/nextclade@alpha
>nextclade --input-fasta error.fasta --output-json error_fasta.json -g nextclade/packages/web/src/algorithms/defaults/sars-cov-2/geneMap.json

Error: in sequence "Canada/BC_37_0-2/2020": geneMap.map is not a function. This sequence will be excluded from further analysis.
```

- Same issue with dockerised CLI tool:

```
>docker pull neherlab/nextclade
>docker run -it --rm -u 1000 --volume="$PWD:/seq" neherlab/nextclade nextclade.js --input-fasta /seq/error.fasta --output-json /seq/error_fasta.json -g /seq/nextclade/packages/web/src/algorithms/defaults/sars-cov-2/geneMap.json

Error: in sequence "Canada/BC_37_0-2/2020": Unknown error. Please note that this sequence will not be included in the results.
```

## Invocations that work
- Succeeds with npm install of current beta release of CLI tool (the older v0.4.0-beta.6)

```
>npm install --global @neherlab/nextclade@beta
>nextclade --input-fasta error.fasta --output-json error_fasta.json -g nextclade/packages/web/src/algorithms/defaults/sars-cov-2/geneMap.json
```

- Succeeds uploading to https://clades.nextstrain.org/ running v0.7.6 beta processes without error 

- Succeeds uploading to local server (deployed via `yarn dev` instructions in repo) running v0.8.0-alpha.6  

- Succeeds running v0.7.5 in CLI without `-g geneMap.json` (docker and npm intsall)

- Succeeds running v0.8.0-alpha.6 CLI without `-g geneMap.json` (npm install)

**Comment by ivan-aksamentov** [comment]
@fmaguire Hi Finlay, thanks for the detailed report!

"Unknown error" is not good. Likely a bug on our side. I'm investigating this immediately.

In order for me to reproduce the issue locally, can you confirm that this is the same sequence as  "EPI_ISL_412965" from GISAID?

**Comment by fmaguire** [comment]
Yeah, the example is `EPI_ISL_412965` downloaded via the GISAID nextstrain sequence file. I had the same issue for every single GISAID sequence I tried from that file for some reason.

The `0.8.0-alpha.6` version error message (`geneMap.map is not a function`) and the fact it works without `-g`, suggests it is something in the geneMap parsing, I'd dive in but think I'd be more of a liability than a help in javascript!

**Comment by ivan-aksamentov** [comment]
@fmaguire I've released 0.8.0-alpha.7 with a candidate fix. Cold you please check if it now works for you? I will backport and release in 0.7.8 then on stable channel.

P.S. Please note that 0.8.x branch is not fully stable and includes some changes to the algorithm, as well as new subclades 20.EU1 and 20.EU2. See [CHANGELOG.md](https://github.com/nextstrain/nextclade/blob/master/packages/web/CHANGELOG.md).

**Comment by fmaguire** [comment]
Totally fixes the issue in the alpha candidate. Thanks @ivan-aksamentov really appreciate your incredibly fast response and all your work on nextclade!

**Comment by ivan-aksamentov** [comment]
@fmaguire Released as 0.7.9 in stable channel, in case you need consistency with other results from 0.7.x version. This is the recommended version.

(there's been a little mishap with the new release script, so version 0.7.8 got skipped)
